app/Models/User.php

```diff
@@ -388,14 +388,37 @@ public static function checkWhenUsernameAvailable($username) : Carbon
 
     public function getUsernameAvailableAt() : Carbon
     {
-        if ($this->group_id !== 2 || $this->user_type === 1) {
+        $playCount = array_reduce(array_keys(Beatmap::MODES), function ($result, $mode) {
```

Comment:
Looks like I forgot to mention that this should use `$this->playCount()` instead.